nilearn/region/AutoExtractor.py

```diff
@@ -0,0 +1,132 @@
+"""
+Auto Extractor
+"""
+
+import numpy as np
+
+from scipy.ndimage.measurements import label
+
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.externals.joblib import Memory
+
+from nilearn._utils import check_niimg
+
+
+def compute_low_threshold(maps, ratio):
+    """
+    Parameters
+    ----------
+    maps: numpy array like of
```

Comment:
how about calling this ```data```as we usually do for non-niimg objects with imaging information. Map might give the impression that is a nibabel object